packages/xdl/src/Project.ts

```diff
@@ -1891,9 +1892,14 @@ export async function startExpoServerAsync(projectRoot: string): Promise<void> {
       extended: true,
     })
   );
-  if ((await Doctor.validateWithNetworkAsync(projectRoot)) === Doctor.FATAL) {
+  if (
+    (ConnectionStatus.isOffline()
+      ? await Doctor.validateLowLatencyAsync(projectRoot)
```

Comment:
this method has a bit of a strange name. i think it should be renamed to `validateHighLatencyAsync` or `validateWithoutNetworkAsync`. i know it wasn't added in this pr but it could be good to clean up. will require updating other callsites ofc